test/test_arhp.rb

```diff
@@ -180,4 +196,16 @@ def assert_action_uses_correct_database(action, sql)
       assert_equal desired_db, current_database(klass)
     end
   end
+
+  def simulate_rails_app_active_record_railties
+    if ActiveRecord.version >= Gem::Version.new('6.0')
+      # Necessary for testing ActiveRecord 6.0 which uses the connection
+      # handlers when clearing query caches across all handlers when
+      # an operation that dirties the cache isinvolved (e.g. create/insert,
```

Comment:
```suggestion
      # an operation that dirties the cache is involved (e.g. create/insert,
```

Reply:
Typo fixed in updated in baca2158b0232376301ee2dfed194bf61063d29b